code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -34,6 +48,146 @@
+		<b>Fire mode:</b> <a href='?src=\ref[src]&fire_mode=\ref[my_laser_tag_game]'>[my_laser_tag_game.fire_mode] </a> <br/>
+		<b>Stun time:</b> <a href='?src=\ref[src]&stun_time=\ref[my_laser_tag_game]'>[my_laser_tag_game.stun_time] </a> <br/>
+		<b>Disable time:</b> <a href='?src=\ref[src]&disable_time=\ref[my_laser_tag_game]'>[my_laser_tag_game.disable_time] </a> <br/>
+		<b><a href='?src=\ref[src]&delete_game=\ref[my_laser_tag_game]'>Delete the game</a></b> <br/>
+		<br/>
+		<b><a href='?src=\ref[src]&edition_done=\ref[my_laser_tag_game]'>Done</a></b>
+	""}
+	user << browse(dat,"window=laser_tag_window2;size=250x250")
+
+/obj/item/clothing/suit/tag/Topic(href, href_list)
+	if(..())
+		return 1
+
+	if (href_list["join_game"])
+		var/datum/laser_tag_game/game = locate(href_list["join_game"])
+		game.handle_new_player(player, usr)
+		my_laser_tag_game = game
+		usr << browse(get_window_text(usr),"window=laser_tag_window;size=500x250")
+		return
+	
+	if (href_list["create_game"])
+		var/datum/laser_tag_game/game = new
+		game.owner = player
+		my_laser_tag_game = game
+		game.name = "[get_first_word(usr.name)]'s game"
+		game.handle_new_player(player, usr)
+		refresh_edit_window(usr, game)
+		usr << browse(get_window_text(usr),"window=laser_tag_window;size=500x250")
+		return
+
+	// Game parametrisation
+	if (href_list["edit_game"])
+		var/datum/laser_tag_game/game = locate(href_list["edit_game"])
+		if (my_laser_tag_game.owner != player)
+			return
+		refresh_edit_window(usr, game)
+		return
+	
+	if (href_list["game_mode"])
+		var/datum/laser_tag_game/game = locate(href_list["game_mode"])
+		var/choices = list(
+			LT_MODE_TEAM,
+			LT_MODE_FFA,
+		)
+		var/choice = input(usr, "Choose the game mode.", "Game mode") as null|anything in choices
+		if (choice)
+			game.mode = choice
+		refresh_edit_window(usr, game)
+		return
+
+	if (href_list["fire_mode"])
+		var/datum/laser_tag_game/game = locate(href_list["fire_mode"])
+		var/choices = list(
+			LT_FIREMODE_LASER,
+			LT_FIREMODE_TASER,
+		)
+		var/choice = input(usr, "Choose the fire mode.", "Fire mode") as null|anything in choices
+		if (choice)
+			game.fire_mode = choice
+		refresh_edit_window(usr, game)
+		return
+	
+	if (href_list["stun_time"])
+		var/datum/laser_tag_game/game = locate(href_list["stun_time"])
+		var/choice = input(usr, "Choose the stun duration.", "Stun duration") as null|num
+		game.stun_time = clamp(choice, 0, 12)
+		refresh_edit_window(usr, game)
+		return
+
+	if (href_list["disable_time"])
+		var/datum/laser_tag_game/game = locate(href_list["disable_time"])
+		var/choice = input(usr, "Choose the disbale duration.", "Disable duration") as null|num
+		game.disable_time = clamp(choice, 0, 30)
+		refresh_edit_window(usr, game)
+		return
+
+	if (href_list["edition_done"])
+		usr << browse(null,"window=laser_tag_window2;size=250x250")
+		return
+
+	if (href_list["delete_game"])
```

Comment:
I feel like this needs some sanity to prevent href exploits

Reply:
Correct, this one doesn't check if the dude deleting it is the owner of the game.
"Regular" href exploits (using it whilst dead, using it remotely, etc) are handled by `if (..()) return`